t/web/house_download.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+require 'test_helper'
+require_relative '../../app'
+
+describe 'house download template' do
+  subject { Nokogiri::HTML(last_response.body) }
+  before { get '/united-states-of-america/senate/download.html' }
+
+  describe 'headings' do
+    let(:banner)       { subject.css('.page-section--grey h3').text }
+    let(:house_header) { subject.css('.country__legislature__header h2').text }
+
+    it 'show the house name in the banner' do
```

Comment:
these read a little weirdly — the "it" form exists so that these can read like normal sentences: e.g. "it should show…" or "it shows…" rather than "it show"